src/cli/init/templates.js

```diff
@@ -0,0 +1,42 @@
+const { prettyProtectedFileContent } = require('../infrastructure/commons');
+
+const pageTemplate = prettyProtectedFileContent(`
+    import manifest from '@manifest';
+    import { CatchAll } from '@kiwi-app/kiwi-nextjs';
+    import ClientComponent from './client';
+    import ServerComponent from './server';
+    
+    const Page = CatchAll(manifest, ClientComponent, ServerComponent);
+
+    export default Page;    
+`);
+
+const clientCodeTemplate = prettyProtectedFileContent(`
+    'use client'
```

Comment:
this 'use client' must be on top of the file, otherwise next does not consider.

maybe you can add another parameter for 'client file' or just create another method that abstracts this behavior